src/components/App/App.js

```diff
@@ -15,23 +14,16 @@ class App extends Component {
     this.state = {
       cities: [],
       searchedCities: [],
-      isLoading: true,
```

Comment:
Good call on pulling loading from state.  I think if the application required more fetch calls then it would be helpful, but we've just got the two!